ADOPTERS.MD

```diff
@@ -10,8 +10,12 @@ If you would like to be included in this table, please submit a comment to [this
 | ------------------ | --------------------------------------- | ------------------- | -------------------------------------------------------------------- |
 | Blue Origin        |                                         | Production          | [link](https://www.blueorigin.com/)                                  |
 | Fulcrum            | Fulcrum                                 | Production          | [link](https://www.fulcrumapp.com/)                                  |
+| Grey Rook          |                                         | Production          | [link](https://www.greyrook.com)                                     |
 | IBM                | IBM Cloud Code Engine                   | Production          | [link](https://cloud.ibm.com/codeengine)                             |
+| Red Hat            | Red Hat OpenShift Serverless            | Generally Available | [link](https://www.openshift.com/learn/topics/serverless)            |
+| Scaleway           |                                         | Public Beta         | [link](https://www.scaleway.com/en)                                  |
 | Tata Communication |                                         | Development         | [link](https://www.tatacommunications.com/solutions/cloud/platforms) |
 | TriggerMesh        |                                         | Production          | [link](https://triggermesh.com)                                      |
 | VMware             | Cloud Native Runtimes for VMware Tanzu  | Generally Available | [link](https://network.tanzu.vmware.com/products/serverless)         |
 | VMware             | VMware Event Broker Appliance           | Development         | [link](https://vmweventbroker.io/)                                   |
+| deepc.ai           |                                         | Production          | [link](https://www.deepc.ai/)                                        |
```

Comment:
How are we thinking about sorting this? Append, alphabetic, or something else? If Alphabetic (which makes sense to me, and looks like you did that), wouldn't this go before Fulcrum?

Reply:
I did :sort in vim and I think it put deepc.ai last because it's lowercase. Thoughts?

Reply:
I'd rather have them alphabetic and not include capitalization in the sorting.

Reply:
updated on the latest commit :) PTAL